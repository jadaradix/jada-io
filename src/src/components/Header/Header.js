import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components'

const StyledHeader = styled.h1`
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
  font-size: 3rem;
  line-height: 2rem;
  font-weight: 800;
  -webkit-font-smoothing: antialiased;
  color: rgb(204, 204, 204);
  line-height: 3rem;
  letter-spacing: -2px;
  // background-color: green;
`;

const Header = ({ children }) => (
  <StyledHeader>
    {children}
  </StyledHeader>
);

Header.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ])
};

Header.defaultProps = {};

export default Header;
