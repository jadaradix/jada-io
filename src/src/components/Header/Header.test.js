import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Header from './Header'

test('renders', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
