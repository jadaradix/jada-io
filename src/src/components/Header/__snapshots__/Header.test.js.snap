// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",sans-serif;
  font-size: 3rem;
  line-height: 2rem;
  font-weight: 800;
  -webkit-font-smoothing: antialiased;
  color: rgb(204,204,204);
  line-height: 3rem;
  -webkit-letter-spacing: -2px;
  -moz-letter-spacing: -2px;
  -ms-letter-spacing: -2px;
  letter-spacing: -2px;
}

<h1
  className="c0"
/>
`;
